DMCompiler/Compiler/DM/DMAST.cs

```diff
@@ -149,6 +148,22 @@ public abstract class DMASTProcStatement : DMASTNode {
         public DMASTProcStatement(Location location)
             : base(location)
         {}
+        /// <summary>
+        /// Gets whether this statement is or contains some <see cref="DMASTProcStatementSet"/>s.<br/>
+        /// This is <see langword="fucked"/> but having the helper in general is actually quite convenient.
+        /// </summary>
+        public bool IsSetStatement {
```

Comment:
This could be made into a generic method
```cs
public bool Is<T>() where T : DMASTProcStatement {
    return (this is T or DMASTAggregate<T>);
}
```
This could be used like `Is<DMASTProcStatementSet>()`